### PR TITLE
Fix/add loading maps and visu

### LINKED ISSUE #129: Add loading to Maps & Visu maps.
# Solve after #127 (there's a branch in this branch [sorry] that solves the problem)

**Is your feature request related to a problem? Please describe.**
Currently, the visualizations options can be changed even when the previous one is not yet loaded. So, to avoid and solve rendering problems and bugs, there should be a loading to keep the users from changing visualizations before the previous one is loaded and rendered.

**Describe the solution you'd like**
Adding a loading feature in those maps should solve the problems.

### PR DESCRIPTION
**Pull request by LazaroQueiroz**
# this PR **MUST** be merged **AFTER** #127. there are some features which depend on the modifications made in that PR.

This PR adds a loading feature to the maps at "Plataforma OCA" at the main page.
Before, users could change visualizations even before the previous map where loaded and/or rendered. It was causing some problems and bugs in the rendering process. Now, there's a loading feature that prevents the user from changing visualizations before the last one is properly loaded/rendered.

## How to test it:
1. go to '/'
2. scroll down to "Plataforma Oca" section
3. try changing the visualization option

## Expected behavior
when changing the visualization, while the map is not fully loaded, there should be a loading icon where usually is the "open/closed eye" is and all the visualization options should have a lower opacity than normal (0.6 instead of 1). Also, when hovering others options while the loading is happening, the cursor must be "not-allowed". When clicking on others visualizations options, the click handler should not be triggered, e.t., there should not be a change of visualization before the loading is finished.

## Screenshots
image

### CODE REVIEW

**Review by abraaovalentim — APPROVED** [review]
Great Work

File: src/components/MapsSection/MapsSection.styles.tsx [hunk 1]
```diff
@@ -77,7 +77,17 @@ export const TagButton = styled.span<{ active?: string }>`
   box-sizing: border-box;
   background-color: #ffffff95;
 
-  cursor: ${({ active }) => active !== "true" && "pointer"};
+  cursor: ${({ active, isLoading }) => {
```
Comment (LeilaFarias) [inline src/components/MapsSection/MapsSection.styles.tsx]: i think we can change this part, just to follow the pattern 
`cursor: ${({ active, isLoading }) => 
  isLoading ? "not-allowed" : active === "true" ? "" : "pointer"
};`

**Review by LeilaFarias — CHANGES_REQUESTED** [review]

File: src/components/MapsSection/MapsSection.styles.tsx (hunk 1 above)
Comment (LazaroQueiroz) [reply src/components/MapsSection/MapsSection.styles.tsx]: done.

**Review by LeilaFarias — APPROVED** [review]
nice! 
image

**Pull request merged**